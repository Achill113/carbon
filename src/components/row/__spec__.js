import React from 'react';
import TestUtils from 'react/lib/ReactTestUtils';
import Row from './index';

describe('Row', () => {
  it('failing test demo', () => {
    expect(1).toEqual(2);
  });
});
