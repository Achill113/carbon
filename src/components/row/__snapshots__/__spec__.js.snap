// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row columnClasses passes the prop to its children 1`] = `
<div
  className="carbon-row carbon-row--gutter-medium carbon-row--columns-0"
/>
`;

exports[`Row columnDivide when columnDivide is true passes columnDivide to its children 1`] = `
<div
  className="carbon-row carbon-row--gutter-medium carbon-row--columns-0"
/>
`;

exports[`Row columns when column number is not passed determines the columns from the number of children 1`] = `
<div
  className="carbon-row carbon-row--gutter-medium carbon-row--columns-4"
>
  <Column />
  <Column />
  <Column />
  <Column />
</div>
`;

exports[`Row columns when column number is passed sets the passed column as the number of columns 1`] = `
<div
  className="carbon-row carbon-row--gutter-medium carbon-row--columns-50"
>
  <Column />
</div>
`;

exports[`Row gutter applies a gutter class 1`] = `
<div
  className="carbon-row carbon-row--gutter-small carbon-row--columns-0"
/>
`;

exports[`Row render renders a parent div with calculated CSS classes 1`] = `
<div
  className="carbon-row carbon-row--gutter-medium carbon-row--columns-0"
/>
`;
